define/define.py

```diff
@@ -0,0 +1,45 @@
+import urllib2
+import json
+import random
+from datamuse.datamuse import Datamuse
+
+def get_rhyme(word):
+    request = Datamuse.get_perfect_rhymes(word)
```

Comment:
I don't think this will actually run? Don't you need to initialize the Datamuse object? That said, I'm not sure there's any actual benefit to Datamuse being a class rather than a group of functions in a module.